### PR TITLE
Minor changes to dev1

### PR DESCRIPTION
**Pull request by Withington**

### CODE REVIEW

File: views/__init__.py [hunk 1]
```diff
@@ -215,10 +215,6 @@ def change_password():
         print 'LOGIN SUCCESS, CHANGING PASSWORD'
         argon_password = argon2.hash(new_password_1)
         db_users = get_db(app.config['DB_NAME_USERS'])
-        #db_users.users.update_one({'user':username},{'$set':{'password':hash}})
-        q={'query':'MATCH (u:User {user: $user}) SET u.password=$password','parameters':{'user':username,'password':password}}
```
Comment (IsmailM) [inline views/__init__.py:219]: @pontikos, just making sure this is fine?

Is there any reason for the clear text password to be stored into the neo4j db?

**Comment by pontikos** [comment]
No the password should be the argon2 hashed password right?

**Comment by IsmailM** [comment]
yep, It seems that while we were testing, we accidentally also added the clear text password to the User node.

We will also need to manually remove these properties from the neo4j db... 

I'm merging this for now, Thanks @Withington 👍

**Comment by Withington** [comment]
Thanks to you both.

**Pull request merged**